pymongo/uri_parser.py

```diff
@@ -34,26 +40,49 @@
 SRV_SCHEME_LEN = len(SRV_SCHEME)
 DEFAULT_PORT = 27017
 
+# List of sub-delimiters as defined in RFC 3986.
+SUBDELIMS = ["!", "$", "&", "'", "(", ")", "*", "+", ",", ";", "="]
```

Comment:
Please rename to be private (ie `_SUBDELIMS`)

Reply:
Done.